spec/integration/integration_spec_helper.rb

```diff
@@ -64,6 +65,48 @@ def normalize_inspect_output(output)
     gsub!(/(\r\033\[K.*?\r\033\[K).*\r\033\[K(.*)/, "\\1\\2") # strip all progress messages but two
 end
 
+def machinery_boxes
+  {
+    "openSUSE_13_2" => "machinery_132",
+    "openSUSE_13_1" => "machinery_131",
+    "Tumbleweed" => "machinery_tumbleweed"
+  }
+end
+
+def boxes
+  {
+    "openSUSE_13_2" => "opensuse132",
+    "openSUSE_13_1" => "opensuse131"
+  }
+end
```

Comment:
It might be nice to have a naming scheme which is systematic enough to not require extra mappings like the ones above. But if it makes things more explicit and readable for now, this is also something we can fix later.